rust/.cargo/config

```diff
@@ -0,0 +1,2 @@
+[build]
+rustflags = ["-Dunsafe_op_in_unsafe_fn"]
```

Comment:
Why this instead of adding `#![deny(unsafe_op_in_unsafe_fn)]` top the top of `rust/src/lib.rs`?

Reply:
I like it here, since it enables it for any new files that may be added.

Reply:
Since `rust/src/lib.rs` is the only source file at the moment, they are equivalent. This approach is arguably better for multi-file projects (like librustzcash) if our policy is going to be to enable this lint in all of our code. So I decided to do it in the same way here as I would do it in librustzcash.

Reply:
Lints applied at the module level apply to every module under the module they are applied on&lang;/wordsalad&rang;. Using `#![deny(LINT)]` in `lib.rs` makes it a crate-wide lint, which for a single crate is equivalent to passing a compiler flag. And I generally prefer placing things like this into the source code when possible, instead of adding compiler flags (which need more care when interacting with the rest of the build system, particularly in multi-language codebases like this one).

Reply:
For multi-crate workspaces with many lints, having to duplicate across `lib.rs` files can become cumbersome, which is something being discussed and worked on in https://github.com/rust-lang/cargo/issues/5034. But that argument does not apply here as this is (and will always be) a single-crate workspace.